google/cloud/aiplatform/training_jobs.py

```diff
@@ -1744,6 +1791,7 @@ def run(
         test_filter_split: Optional[str] = None,
         predefined_split_column_name: Optional[str] = None,
         timestamp_split_column_name: Optional[str] = None,
+        enable_web_access: Optional[bool] = False,
```

Comment:
Since `enable_web_access` defaults to `False` it seems that this doesn't need to be `Optional`.